### PR TITLE
Meet the artists

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

File: contents/news/meet-the-artists.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+----
+template: pages/news-item.html.njk
+filename: :file/index.html
+
+title: Meet the Artists
+date: 2018-05-16
+----
+
+We’re excited to introduce you to [some of the artists](/speakers/#artists) that will make our 2018 event even more special. Some joined us through the [Call for Artists](https://twitter.com/jsconfeu/status/920317478177087488) and others are familiar faces–but we can't tell you about everyone yet, because that would give too much away 🤣
```
Comment (oe) [inline contents/news/meet-the-artists.md]: s/our 2018 event/JSConf 2018

Comment (cramforce) [reply contents/news/meet-the-artists.md]: Done. Thanks

File: contents/news/meet-the-artists.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+----
+template: pages/news-item.html.njk
+filename: :file/index.html
+
+title: Meet the Artists
+date: 2018-05-16
+----
+
+We’re excited to introduce you to [some of the artists](/speakers/#artists) that will make JSConf 2018 event even more special. Some joined us through the [Call for Artists](https://twitter.com/jsconfeu/status/920317478177087488) and others are familiar faces–but we can't tell you about everyone yet, because that would give too much away 🤣
```
Comment (janl) [inline contents/news/meet-the-artists.md]: make *the* JSConf 2018 or make *our* or something

Comment (cramforce) [reply contents/news/meet-the-artists.md]: Ah, yes. I implemented @oe's feedback badly :)

**Pull request merged**